### ISSUE TITLE
typedef documentation is wrong.

### ISSUE THREAD

**Issue by DanielBocksteger** [onevcat/VVDocumenter-Xcode#91]
If i type my vvdocumenter trigger-code above an typedef in xcode this is the result...
```
/*!
 @brief  <#Description#>
 */
typedef NS_ENUM(NSUInteger, SPXMLExportError){
    /*!
     @brief  <#Description#>
     */
    SPXMLExportErrorUnknown,
    /*!
     @brief  <#Description#>
     */
    SPXMLExportErrorKeinFehler,
    /*!
     @brief  <#Description#>
     */
    SPXMLExportErrorDBFehler,
};
```

Looking at my html this is wrong. 

the document-comment should look like
```
/*!
 @brief  Fehler-typ Definition für den XML export
 
 @constant SPXMLExportErrorUnknown      Unbekannter fehler
 @constant SPXMLExportErrorKeinFehler   Fehlerfrei
 @constant SPXMLExportErrorDBFehler     Datenbank fehler
 
 */
```

**Comment by onevcat** [comment]
Hi, appledoc and earlier version of Xcode is using the the first style (inline doc for enum), so VVDocumenter follows it. The only problem is the `@brief` tag. I will fix it soon.

**Comment by onevcat** [comment]
Should be fixed in #121
